storage/client/queries/queries.go

```diff
@@ -286,31 +286,82 @@ const (
 			ORDER BY id DESC`
 
 	ValidatorsData = `
+		WITH
+		-- Find all self-delegations for all accounts with active delegations.
+		self_delegations AS (
+			SELECT delegatee AS address, shares AS shares
+				FROM chain.delegations
+				WHERE delegator = delegatee
+				GROUP BY delegator, delegatee
+		),
+		-- Compute the number of delegators for each validator.
```

Comment:
```suggestion
		-- Compute the number of delegators for each delegatee.
```

Minor nit, since this computes number of delegators for every account, not just validators.